keps/sig-security/3203-auto-refreshing-official-cve-feed/README.md

```diff
@@ -0,0 +1,690 @@
+  - [ ] (R) Minimum Two Week Window for GA e2e tests to prove flake free
+- [ ] (R) Graduation criteria is in place
+  - [ ] (R) [all GA Endpoints](https://github.com/kubernetes/community/pull/1806) must be hit by [Conformance Tests](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/conformance-tests.md)
+- [ ] (R) Production readiness review completed
+- [ ] (R) Production readiness review approved
+- [ ] "Implementation History" section is up-to-date for milestone
+- [ ] User-facing documentation has been created in [kubernetes/website], for publication to [kubernetes.io]
+- [ ] Supporting documentation—e.g., additional design documents, links to mailing list discussions/SIG meetings, relevant PRs/issues, release notes
+
+<!--
+**Note:** This checklist is iterative and should be reviewed and updated every time this enhancement is being considered for a milestone.
+-->
+
+[kubernetes.io]: https://kubernetes.io/
+[kubernetes/enhancements]: https://git.k8s.io/enhancements
+[kubernetes/kubernetes]: https://git.k8s.io/kubernetes
+[kubernetes/website]: https://git.k8s.io/website
+
+## Summary {#summary}
+
+Currently it is not possible to filter for issues or PRs that are related to CVEs announced by kubernetes. This KEP addresses this concern by labelling this issues or PRs with the new label **official-cve-feed** using the automation. The in-scope issues are the closed issues for which there is a CVE ID and is officially announced as a Kubernetes CVE by SRC in the past.
+
+
+## Motivation {#motivation}
+
+With the growing number of eyes on Kubernetes, the number of CVEs related to Kubernetes have increased. Although most CVEs are regularly fixed that directly or indirectly or transitively impact Kubernetes, there is no single place to programmatically subscribe or pull the data of fixed CVEs, for the end users of Kubernetes. Current options are either broken or incomplete.
+
+
+An auto-refreshing CVE feed will allow end users to programmatically fetch the list of CVEs and allow them to get up to date information from kubernetes community.
+
+
+### Goals {#goals}
+
+
+Create a periodically auto-refreshing list of official Kubernetes CVEs
+
+### Non-Goals
+
+- Triage and vulnerability disclosure: This will continue to be done by SRC
+- CVEs that are identified in build time dependencies and container images. Only official CVEs announced by SRC will be published in the feed.
+
+### User Stories (Optional)
+
+<!--
+Detail the things that people will be able to do if this KEP is implemented.
+Include as much detail as possible so that people can understand the "how" of
+the system. The goal here is to make this feel real for users without getting
+bogged down.
+-->
+
+#### Story 1
+
+As a K8s end user, I want a list of CVEs with relevant information that I can fetch programmatically, so I can understand when new CVEs are announced
```

Comment:
Maybe also a story about acceding the HTML rendered list in a browser, and trusting that said page is current?

If that's not in scope, let's list it as a non-goal for _this_ KEP and explain that it's covered elsewhere. I hope it is in scope though.

Reply:
Added